src/dnstest_harness.erl

```diff
@@ -11,7 +11,10 @@
 -export([init/1, handle_call/3, handle_cast/2]).
 
 % Internal API
--export([send_request/3]).
+-export([send_request/4]).
+
+% Helper functions for dnskey process (RSA)
+-export([decode_dnskey_public_key/1, compute_base64_key/1]).
```

Comment:
These look like protocol specific knowledge, shouldn't they belong to `dns_erlang` maybe?

Reply:
It probably makes sense to add them as utility functions. I was looking for something to help with getting the data prepared so that we can move forward with the test suite.